server/tilesource/base.py

```diff
@@ -55,6 +69,55 @@ def getTile(self, x, y, z):
     def getTileMimeType(self):
         return 'image/jpeg'
 
+    def getThumbnail(self, width=None, height=None, **kwargs):
```

Comment:
Can we document the available ``kwargs`` here? We don't even need detail, but a comprehensive list is important.